tests/x11regressions/gnomecase/login_test.pm

```diff
@@ -29,7 +29,11 @@ sub run () {
 
     assert_screen "generic-desktop";
     $self->auto_login_alter;
+    my $ov = get_var('NOAUTOLOGIN');
+    set_var('NOAUTOLOGIN', '');
     reboot_gnome;
+    wait_boot;
+    set_var('NOAUTOLOGIN', $ov);
```

Comment:
It's not very obvious what this whole construct is for. can this be moved into wait_boot with a config parameter or add some comments in the code here?